runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -71,7 +72,7 @@ def __init__(
         fn: Optional[Callable] = None,
         fn_pointers: Optional[tuple] = None,
         name: Optional[str] = None,
-        env: Optional[list[str] or str] = None,
+        reqs: Optional[list[str]] = None,
         env_vars: Optional[dict] = None,
```

Comment:
can we extract `env_vars` from the env object instead of breaking it out as a separate input to the factory